py/server/deephaven/experimental/ema.py

```diff
@@ -0,0 +1,106 @@
+#
+#     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""This modules allows users to define exponential moving averages that can be used in the Deephaven query language."""
+import numpy
+import jpy
+
+_JAbstractMAType = jpy.get_type('io.deephaven.numerics.movingaverages.AbstractMa$Type')
+_JAbstractMATMode = jpy.get_type('io.deephaven.numerics.movingaverages.AbstractMa$Mode')
+_JBadDataBehavior = jpy.get_type('io.deephaven.numerics.movingaverages.ByEma$BadDataBehavior')
+_JTimeUnit = jpy.get_type('java.util.concurrent.TimeUnit')
+_JByEmaSimple = jpy.get_type('io.deephaven.numerics.movingaverages.ByEmaSimple')
+_JEma = jpy.get_type('io.deephaven.numerics.movingaverages.Ema')
+_JEmaArray = jpy.get_type('io.deephaven.numerics.movingaverages.EmaArray')
+_JExponentiallyDecayedSum = jpy.get_type('io.deephaven.numerics.movingaverages.ExponentiallyDecayedSum')
+
+
+def _convertEnumBehavior(value, enumType):
+    if value is None or isinstance(value, enumType):
+        return value
+    elif hasattr(enumType, value):
+        return getattr(enumType, value, None)
+    return value
+
+
+def ByEmaSimple(nullBehavior, nanBehavior, mode, timeScale, timeUnit, type=None):
```

Comment:
There are a lot of non-pep8 names.  Should these be changed for this case, or not?  I can agree to not, since this is in experimental and it is just a stop gap, but making it standards compliant is also a thing.  Just a note.  I don't have a view in this case.

Reply:
Yeah, it was a deliberate decision to make it obvious that these are not  fully wrapped final version.